pennylane/templates/subroutines/single_excitation_op.py

```diff
@@ -23,7 +23,7 @@
 
 
 @template
-def SingleExcitationUnitary(weight, wires=None):
+def SingleExcitationUnitary(weight, wires_rp=None):
```

Comment:
@agran2018: note that this should remain `wires`, for consistency, and compatibility with the `VQE` and `Collections` module, which both assume the provided ansatz has an argument named `wires`.

Better to clarify the relationship between `wires` and `r,p` in the docstring, rather than the signature.